packages/manager/src/features/LoadBalancers/LoadBalancerDetail/LoadBalancerServiceTargets.tsx

```diff
@@ -0,0 +1,185 @@
+import { useLoadBalancerServiceTargetsQuery } from 'src/queries/aglb/serviceTargets';
+
+import type { Filter } from '@linode/api-v4';
+
+const PREFERENCE_KEY = 'loadbalancer-service-targets';
+
+export const LoadBalancerServiceTargets = () => {
+  const { loadbalancerId } = useParams<{ loadbalancerId: string }>();
+
+  const [query, setQuery] = useState<string>();
+
+  const pagination = usePagination(1, PREFERENCE_KEY);
+
+  const { handleOrderChange, order, orderBy } = useOrder(
+    {
+      order: 'desc',
+      orderBy: 'label',
+    },
+    `${PREFERENCE_KEY}-order`
+  );
+
+  const filter: Filter = {
+    ['+order']: order,
+    ['+order_by']: orderBy,
+  };
+
+  // If the user types in a search query, filter results by label.
+  if (query) {
+    filter['label'] = { '+contains': query };
+  }
+
+  const { data, error, isLoading } = useLoadBalancerServiceTargetsQuery(
+    Number(loadbalancerId),
+    {
+      page: pagination.page,
+      page_size: pagination.pageSize,
+    },
+    filter
+  );
+
+  if (isLoading) {
+    return <CircleProgress />;
+  }
+
+  return (
+    <>
+      <Stack
+        alignItems="flex-end"
+        direction="row"
+        flexWrap="wrap"
+        gap={2}
+        mb={2}
+        mt={1.5}
+      >
+        <TextField
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="Clear"
+                  onClick={() => setQuery('')}
+                  size="small"
+                  sx={{ padding: 'unset' }}
+                >
+                  <CloseIcon
+                    color="inherit"
+                    sx={{ color: '#aaa !important' }}
+                  />
+                </IconButton>
+              </InputAdornment>
+            ),
+          }}
+          hideLabel
+          label="Filter"
+          onChange={(e) => setQuery(e.target.value)}
+          placeholder="Filter"
+          style={{ minWidth: '320px' }}
+          value={query}
+        />
+        <Box flexGrow={1} />
+        <Button buttonType="primary">Create Service Target</Button>
+      </Stack>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableSortCell
+              active={orderBy === 'label'}
+              direction={order}
+              handleClick={handleOrderChange}
+              label="label"
+            >
+              Label
+            </TableSortCell>
+            <TableCell>Endpoints</TableCell>
+            <Hidden smDown>
+              <TableCell>Algorithm</TableCell>
+            </Hidden>
+            <Hidden mdDown>
+              <TableCell>Certificate</TableCell>
+            </Hidden>
+            <Hidden lgDown>
+              <TableCell>Health Checks</TableCell>
+            </Hidden>
+            <TableCell></TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {error && <TableRowError colSpan={3} message={error?.[0].reason} />}
```

Comment:
```suggestion
          {error && <TableRowError colSpan={5} message={error?.[0].reason} />}
```
Our error row is just a little short.

![Screenshot 2023-08-23 at 2 08 26 PM](https://github.com/linode/manager/assets/114685994/9759a9b3-ccc8-40eb-b6fb-e530033e8747)